vncd.cpp

```diff
@@ -322,12 +326,24 @@ void parseArguments(int argc, char **argv)
 	}
 }
 
+class RotationHelper : public com::rotationhelper::BnRotationHelper {
+        ::android::binder::Status setRotation(int32_t rotation) {
+        L("Set new rotation: %d\n", rotation);
+        rotate(rotation);
```

Comment:
This function doesn't exist. I think the correct function is called rotateDisplay or rotateScreen. How ever if I comment this line I got different build errors.  

```
ld.lld: error: undefined symbol: closeDisplay()
>>> referenced by vncd.cpp:66 (external/vncserver/vncd.cpp:66)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(closeVncServer(int))
>>> referenced by vncd.cpp:66 (external/vncserver/vncd.cpp:66)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(main)

ld.lld: error: undefined symbol: closeFlinger()
>>> referenced by vncd.cpp:67 (external/vncserver/vncd.cpp:67)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(closeVncServer(int))
>>> referenced by vncd.cpp:67 (external/vncserver/vncd.cpp:67)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(main)

ld.lld: error: undefined symbol: initFlinger()
>>> referenced by vncd.cpp:324 (external/vncserver/vncd.cpp:324)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(main)

ld.lld: error: undefined symbol: initDisplay()
>>> referenced by vncd.cpp:325 (external/vncserver/vncd.cpp:325)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(main)

ld.lld: error: undefined symbol: getScreenRotation()
>>> referenced by vncd.cpp:364 (external/vncserver/vncd.cpp:364)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(main)

ld.lld: error: undefined symbol: readBuffer(unsigned int*)
>>> referenced by vncd.cpp:372 (external/vncserver/vncd.cpp:372)
>>>               out/target/product/rpi4/obj/EXECUTABLES/vncd_intermediates/vncd.o:(main)
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
09:51:12 ninja failed with: exit status 1
```  
@IvanSlaev Please fix this or tell me how to build your solution correctly.